src/components/Modal/style.scss

```diff
@@ -0,0 +1,80 @@
+.modal-wrapper {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.7);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 999;
+    min-width: 400px;
+    
+    .confirmation-modal-container {
+        position: relative;
+        background-color: #F0F0F0;
+        display: flex;
+        flex-direction: column;
+        width: 30%;
+        height: 35%;
+        justify-content: flex-start;
+        border-radius: 25px;
+
+        .button-container {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            position: absolute;
+            padding-bottom: 30px;
+            width: 100%;
+            left: 0;
+        }
+    }
+
+    .modal-container {
+        position: absolute;
+        background-color: #F0F0F0;
+        display: flex;
+        flex-direction: column;
+        width: 70%;
+        height: 70%;
+        justify-content: flex-start;
+        border-radius: 25px;
+
+        .modal-table {
+            overflow-y: scroll;
+            height: 30vh;
+
+            img {

```

Comment:
inconsistent indentation. @nkansal96 is eslint supposed to catch this?

Reply:
`prettier` is supposed to (the general rule for tabs/spacing is 2-space spaces) but I forgot to add the configuration file. @howardwang15 you need to add a file called `prettier.config.js` at the root of this repository with the following contents:

```js
module.exports = {
  printWidth: 120,
  tabWidth: 2,
  useTabs: false,
  semi: true,
  singleQuote: false,
  trailingComma: "es5",
  bracketSpacing: true,
  jsxBracketSameLine: true,
  arrowParens: "avoid"
};
```

Reply:
then you need to make an edit in this file and let `prettier` lint it when you `git commit`